api/HTMLFormControlsCollection.json

```diff
@@ -23,10 +23,10 @@
             "version_added": false
           },
           "opera": {
-            "version_added": true
+            "version_added": "15"
```

Comment:
http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8800 in Opera 12.16 shows `form.elements` was a plain `HTMLCollection` there, so this seems right too.